user/views.py

```diff
@@ -2,10 +2,7 @@
 from json     import JSONDecodeError
 from jwt      import DecodeError
 
-import bcrypt
-import json
-import jwt
-import re
+import bcrypt, json, jwt, re
```

Comment:
제가 보기에 이게 더 깔금해 보이는 것 같은데 어떠신가요?
```suggestion
import bcrypt, json, jwt, re

from json     import JSONDecodeError
from jwt      import DecodeError
```

Reply:
저는 위에께 더 깔끔해보이긴 하는데... 일단 바꾸겠습니다!ㅎㅎㅎ😂

Reply:
```python
import (
    jwt,
    bcrypt,
    joson,
    re
)
```
이 방식이 컨벤션이라고 합니다. 
제가 잘 못 전달해 드렸습니다. 수정 부탁드릴게요.

Reply:
넵 감사합니다!!